src/asmcnc/apps/systemTools_app/screens/screen_support_menu.py

```diff
@@ -148,8 +148,4 @@ def update_font_size(self, value):
         elif len(value.text) > 15: 
             value.font_size = self.default_font_size - 2
         if len(value.text) > 19: 
-            value.font_size = self.default_font_size - 4
-        if len(value.text) > 22: 
-            value.font_size = self.default_font_size - 5
-        if len(value.text) > 25: 
-            value.font_size = self.default_font_size - 6
```

Comment:
were these just unnecessary?

Reply:
Just had a look and it all looks fine, I'm guessing they were just making the text too small to be easily legible